instances/latest/contentTypes/application_vnd.nest-desktop.model.v4+json.jsonld

```diff
@@ -0,0 +1,19 @@
+{
+    "@context": {
+      "@vocab": "https://openminds.om-i.org/props/"
+    },
+    "@id": "https://openminds.om-i.org/instances/contentTypes/application_vnd.nest-desktop.model.v4+json",
+    "@type": "https://openminds.om-i.org/types/ContentType",
+    "dataType": null,
+    "description": "NEST Desktop model specification version 4",
+    "displayLabel": null,
+    "fileExtension": [
+      ".json"
+    ],
+    "name": "application/vnd.nest-desktop.model.v4+json",
```

Comment:
Should the version not come before ?
vnd.nest-desktop.v4.model+json

Reply:
I thought about that, but v4 of a format specification doesn't necessarily correspond to v4 of the software (in this case it does, but as a general rule I think the version spec should come at the end (just before any "+" modifiers)

Reply:
okay. we may revisit the existing content types. I don't think this was done consistently as you suggested (but I'm totally okay with that suggestion)